docs/docs/guides/tracking/tracing.mdx

```diff
@@ -104,7 +104,19 @@ instance.my_method.call(instance, "World")
 
 #### Call Display Name
 
-Sometimes you may want to override the display name of a call. You can achieve this in one of three ways:
+Sometimes you may want to override the display name of a call. You can achieve this in one of four ways:
+
+0. Change the display name at the time of calling the op:
+
+```python showLineNumbers
+result = my_function("World", __weave={"display_name": "My Custom Display Name"})
+```
+
+:::note
+
+Using the `__weave` dictionary sets the call display name which will take precedence over the Op display name.
```

Comment:
nit: Using the `__weave` dict to set the call display name will take precedence over the Op's set display name